src/popover/popover.ts

```diff
@@ -39,12 +48,16 @@ let nextId = 0;
     :host.bs-popover-left .arrow, :host.bs-popover-right .arrow {
       top: 50%;
     }
-  `]
+  `],
+  animations: [trigger(
+      'fade',
+      [transition('void => enter', useAnimation(fadeIn)), transition('enter => exit', useAnimation(fadeOut))])]
 })
```

Comment:
I think this is easier to read when it's split up a bit:

```ts
animations: [
  trigger('fade', [
    transition('void => enter', useAnimation(fadeIn)), 
    transition('enter => exit', useAnimation(fadeOut))
  ]),
]
```

Not sure if you agree :)

Reply:
I do, but this is the code style enforced by clang format. Maybe there's an option to configure it to be prettier in this instance?